MPU/src/pedals.cpp

```diff
@@ -45,6 +45,16 @@ FaultStatus_t Pedals::readAccel()
 	//Calculate the correct amount of torque to command using precentage pressed value
 	appliedTorque = calcTorque(multiplier);
 
+	if (drive_state == PIT || drive_state == REVERSE) {
+		int16_t mph = (motorController->getMotorSpeed() * 0.013048225);
+		if (mph > 5) {
+			appliedTorque = 0;
```

Comment:
This is close but in reality might make people frustrated, I think we should have some sort of torque curve that we derate torque until we get to 5mph. Setting this to 0 at a hard threshold will result in the motor giving a lot of torque and then none since we'll be edging on the threshold (i.e. we'll go from 4.9mph where we give fatty torque and then 5mph where we give none and then we'll drop below and give fatty torque, etc)